internal/cmd/run/run.go

```diff
@@ -135,6 +136,7 @@ func Command() *cobra.Command {
 	cmd.PersistentFlags().StringVar(&gFlags.sauceAPI, "sauce-api", "", "Overrides the region specific sauce API URL. (e.g. https://api.us-west-1.saucelabs.com)")
 	cmd.PersistentFlags().StringVar(&gFlags.selectedSuite, "select-suite", "", "Run specified test suite.")
 	cmd.PersistentFlags().BoolVar(&gFlags.testEnvSilent, "test-env-silent", false, "Skips the test environment announcement.")
+	cmd.PersistentFlags().BoolVar(&gFlags.disableSegment, "disable-segment", false, "Disable segment metrics collections.")
```

Comment:
So if we switch our metrics collection vendor, we have to change the flag name as well? 😅

This particular flag name should be implementation unaware. You are just trying to skip usage metrics.

Reply:
ok! I'll update that